model-engine/model_engine_server/infra/gateways/resources/k8s_endpoint_resource_delegate.py

```diff
@@ -1505,6 +1996,67 @@ async def _get_resources(
             image=common_params["image"],
             num_queued_items=None,
         )
+
+        return infra_state
+
+    async def _get_resources_from_lws_type(
+        self,
+        endpoint_id: str,
+        deployment_name: str,
+        endpoint_type: ModelEndpointType,
+        lws_config,
+        config_maps: List,
+    ) -> ModelEndpointInfraState:
+        k8s_resource_group_name = _endpoint_id_to_k8s_resource_group_name(endpoint_id)
+
+        config_maps = await self._get_config_maps(
```

Comment:
we're query here again? did you mean to remove?

Reply:
yes oops I had commented it out, and thought I needed it and uncommented it by mistake